tower.yml

```diff
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  samplesheet.csv:
+    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
```

Comment:
it would be nice to details more the small files created by the pipeline here.
cf https://github.com/nf-core/rnaseq/blob/master/tower.yml / https://github.com/nf-core/sarek/blob/master/tower.yml